### ISSUE TITLE
Pasting multi-line cells from Excel inserts a carriage return and spaces

### ISSUE THREAD

**Issue by jonw-bb** [handsontable/handsontable#6258]
### Description

Pasting a cell from Excel which contains multi-line text adds a carriage return after the first word of the first line. It also inserts extra spaces. While you can't see it in the screenshot below, if you edit the cell in handsontable, you will see spaces added to the beginning of each new line.

The pasted text in handsontable should retain the same formatting as Excel.

Text copied from Excel:
![excel_multi_line](https://user-images.githubusercontent.com/20171353/64629597-64987880-d3eb-11e9-84ad-0671ba524e3f.png)

Pasted into Handsontable:
![hot_multi_line](https://user-images.githubusercontent.com/20171353/64629686-93aeea00-d3eb-11e9-8c43-6a1495e39c12.png)

I believe this issue reported previously is exactly the same problem
 - https://forum.handsontable.com/t/copy-pasting-value-lists-sometimes-inserts-extra-space-or-carriage-return/3474

I debugged the code in handsontable.full.js where it performs paste actions.

```
var textHTML = event.clipboardData.getData('text/html');
```

Here is an excerpt from the HTML from the clipboard after copying from Excel:

```
<table border=0 cellpadding=0 cellspacing=0 width=189 style='border-collapse:
 collapse;width:142pt'>
 <col width=189 style='mso-width-source:userset;mso-width-alt:6058;width:142pt'>
 <tr height=149 style='height:112.0pt'>
<!--StartFragment-->
  <td height=149 class=xl65 width=189 style='height:112.0pt;width:142pt'>Line
  1<br>
    Line 2<br>
    <br>
    Line 3<br>
    <br>
    Start of a paragrah<br>
    <br>
    Another paragraph</td>
<!--EndFragment-->
 </tr>
</table>
```

Notice how after the first "Line" word, there is a carriage return to the next line. This seems to reflect in the pasted text in handsontable. You seem to be using both a carriage return and the <br> tags.

Commenting some code as suggested here resolves the issue - https://github.com/handsontable/handsontable/issues/5647#issuecomment-466409978 But I assume the commented code is needed for other use cases.

Also please note this problem does not occur if copying from Google Sheets, since the HTML in the clipboard is a different format.

### Steps to reproduce

1. Open Microsoft Excel
2. In an empty cell, type some text and have it so its multi-line (for new lines - Alt+Enter on Windows, Ctrl+Option+Enter on MacOs). Make sure the first line has more than one word.
3. Come out of edit mode. Select the cell and copy it to clipboard
4. In Handsontable, paste into a cell.

### Demo

https://jsfiddle.net/handsoncode/8ffpsqt6/

Not possible to demo without using Excel. You can download a trial from Microsoft - https://products.office.com/en-gb/excel

### Your environment
* Handsontable version: 7.1.1
* Browser Name and version: Chrome 76.0.3809.132
* Operating System: MacOS High Sierra 10.13.6
* Excel version: 16.28 (MacOS)

**Comment by jonw-bb** [comment]
I am aware of https://github.com/handsontable/handsontable/issues/5647 but it has _not_ resolved this issue.

**Comment by aninde** [comment]
@jonw-bb this issue will be fixed in `7.3.0`. We plan to release it on next week.
I test it on your set of multi-line cell pasted from Excel 16.31 and it works.

<img width="113" alt="Screenshot 2019-11-15 at 11 23 26" src="https://user-images.githubusercontent.com/30186818/68936354-a56d8e80-079a-11ea-86bd-1d18365bfc9d.png">


![Screenshot 2019-11-15 at 11 27 00](https://user-images.githubusercontent.com/30186818/68936469-ecf41a80-079a-11ea-8add-ae7c0c8a8075.png)



**Demo**
v. `7.3.0` https://jsfiddle.net/aninde/4gn2q0ey/

**Comment by AMBudnik** [comment]
Works well on Windows Chrome 78 and FF 70

**Comment by aninde** [comment]
The latest build of `7.3.0 v.`(6.12) -is still fixed during testing on Chrome 78, Firefox 70, Safari 13, Microsoft Dev Edge / Mac OS Catalina and IE, Edge, Edge Dev / Windows 11. You can paste multi-line paragraphs from Excel, Numeric, Google Spreadsheet.
https://jsfiddle.net/aninde/nvohepzw/

**Comment by AMBudnik** [comment]
Hey @jonw-bb 

we just released v 7.3.0 where we provide a fix for this issue. I hope that it will also solve the same issue in your project after updating.
